lib/graphiti/sideload.rb

```diff
@@ -411,5 +411,20 @@ def infer_resource_class
     def namespace_for(klass)
       Util::Class.namespace_for(klass)
     end
+
+    def evaluate_flag(flag)
+      return false if flag.blank?
+
+      case flag.class.name
+      when "Symbol"
+        resource.send(flag)
+      when "Proc"
+        flag.call
```

Comment:
This isn't evaluating in the context of the resource, you'll need `instance_eval` (and spec that tests it)

Reply:
@richmolj I took the approach of passing the resource as the explicit context to the blocks through `instance_exec`. Would love your thoughts/feedback :)

Reply:
Works for me!